bin/build_nr_scripts.sh

```diff
@@ -0,0 +1,17 @@
+source .env
```

Comment:
I don't think this worked for me? Or at least it wasn't visible to the ruby process? My JS file ended up with a bunch of blank secrets after I updated the `.env`

I tried `<%= require 'dotenv/load' %>` to the top of the ERB and that made it work for me